pom.xml

```diff
@@ -69,14 +69,25 @@
         <copyrightNotice>Copyright ${copyrightYear} Micro Focus or one of its affiliates.</copyrightNotice>
         <enforceBannedDependencies>true</enforceBannedDependencies>
         <enforceCorrectDependencies>true</enforceCorrectDependencies>
+        <jersey.version>2.37</jersey.version>
     </properties>
 
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-annotations</artifactId>
-                <version>2.9.9</version>
+                <version>2.14.0-rc1</version>
```

Comment:
I have made this optional so that the version wouldn't be changed in restadapter-filesystem

Reply:
They are not optional dependencies.  They are test-scope dependencies.

Reply:
`jackson-annotations` is explictly declared with 2.13.4 in restadapter-filesystem so changing this version doesn't impact it

Reply:
I'm not very keen on the fact that we are pointing at a release candidate rather than an actual release.  Where are you getting that Jersey requires `2.14.0-rc1`?